lib/algolia/client.rb

```diff
@@ -490,6 +490,12 @@ def perform_request(session, url, method, data, request_options)
       return JSON.parse(response.content)
     end
 
+    def format_user_agent(custom)
```

Comment:
It works well, but I think we could simply do a constant for UA to avoid useless string allocations.
Something like:
```
[...]
'User-Agent' => USER_AGENT + data[:user_agent].to_s
[...]
```